<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaPrincipal</class>
 <widget class="QMainWindow" name="TelaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Compilador</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelCondigoFonte">
    <property name="geometry">
     <rect>
      <x>341</x>
      <y>53</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Código Fonte</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCodigoIntermediario">
    <property name="geometry">
     <rect>
      <x>342</x>
      <y>227</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Analise Lexica</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditCodigoFonte">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>77</y>
      <width>770</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNovo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>12</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Novo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAbrir">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>12</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Abrir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSalvar">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>14</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Salvar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCompilar">
    <property name="geometry">
     <rect>
      <x>528</x>
      <y>12</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Compilar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLimpar">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>12</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditCodigoIntermediario">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>250</y>
      <width>770</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelCodigoCompilado">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>403</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Analise Sintática</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditCodigoCompilado">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>427</y>
      <width>770</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
